_papers/wosc7/p4.md

```diff
@@ -8,11 +8,11 @@ authors:
     affiliation: Zurich University of Applied Sciences
   - name: Josef Spillner
     affiliation: Zurich University of Applied Sciences
-article_link: false
-presentation_filename: false
+article_link: https://doi.org/10.1145/3493651.3493669
+presentation_filename: wosc7-faasfusion-slides.pdf
 video:
   lightning_youtube_id: false
-  talk_youtube_id: false
+  talk_youtube_id: https://www.youtube.com/watch?v=dlttrJKBoZU
```

Comment:
Please replace the youtube URL with just the video ID, in your case the video ID is `dlttrJKBoZU`.

Reply:
Done.